components/renku_data_services/k8s/clients.py

```diff
@@ -18,11 +18,11 @@
 
 from renku_data_services.errors import errors
 from renku_data_services.k8s.client_interfaces import K8sCoreClientInterface, K8sSchedudlingClientInterface
+from renku_data_services.k8s.models import APIObjectInCluster
```

Comment:
Good catch, given we now explicitly instantiate it in this file, this is indeed required.